tests/attestation_plugin/host/host.c

```diff
@@ -40,6 +41,50 @@ void host_verify(
         TEST_CLAIMS_SIZE);
 }
 
+#define DATAFILE_LICENSE_HEADER_SIZE 81
+void read_file_to_buffer(
```

Comment:
Can you make this function return `oe_result_t` and return non OE_OK on failing conditions?